ui/components/app/assets/util/parseAssetId.ts

```diff
@@ -0,0 +1,63 @@
+export type ParsedAssetId = {
+  namespace: string; // Namespace (e.g., eip155, solana, bip122)
+  chainId: string; // Full chain ID (namespace + blockchain ID)
+  assetNamespace: string; // Asset namespace (e.g., slip44, erc20, token, ordinal)
+  assetReference: string; // Asset reference (on-chain address, token identifier, etc.)
+};
+
+export const parseAssetId = (assetId: string): ParsedAssetId => {
```

Comment:
instead of string for the type of assets can we use from `CaipAssetType` from  '@metamask/keyring-api'

Reply:
Yeah absolutely, was looking for this. I take a look at the module.